official/transformer/v2/transformer_main_test.py

```diff
@@ -75,23 +85,26 @@ def test_train_static_batch(self):
     t = tm.TransformerTask(FLAGS)
     t.train()
 
+  @unittest.skipUnless(tf.test.is_built_with_cuda(), 'requires GPU')
   def test_train_1_gpu_with_dist_strat(self):
-    FLAGS.distribution_strategy = "one_device"
+    FLAGS.distribution_strategy = 'one_device'
     t = tm.TransformerTask(FLAGS)
     t.train()
 
+  @unittest.skipUnless(tf.test.is_built_with_cuda(), 'requires GPU')
   def test_train_2_gpu(self):
-    FLAGS.distribution_strategy = "mirrored"
+    FLAGS.distribution_strategy = 'mirrored'
     FLAGS.num_gpus = 2
```

Comment:
while you're here, would you mind adding this extra check so that it doesn't fail when there are <2 GPUs? (this will be useful on the internal GPU test where we have 1 GPU only. 
```
if context.num_gpus() < 2:
 self.skipTest(....)
```